use-symfony-reproducer.md

```diff
@@ -0,0 +1,166 @@
+# Using a Symfony Reproducer
+
+***TIP
+This is part 2 of a 2-part _Reproducer_ series. If you haven't already, check
+out [Part 1](https://symfonycasts.com/blog/symfony-reproducer) first.
+***
+
+Ok, so you've created a reproducer app for a bug you've found in Symfony. For
+this article, we'll assume the reproducer you've created in part 1 is located
+at `~/my-reproducer`.
+
+## Forking the Symfony Repository
+
+The first thing we'll do is _fork_ the [`symfony/symfony`](https://github.com/symfony/symfony)
+repository on GitHub. This will give us our own copy of the Symfony repository
+where we can work.
+
+1. Visit https://github.com/symfony/symfony and click the _Fork_
+   button at the top of the page.
+2. Choose your personal GitHub account as the owner and keep the default name
+   of `symfony`.
+3. Leave all the other options as is and click _Create Fork_. You should now be on https://github.com/your-username/symfony.
+4. Wait a few seconds for the fork to be created.
+5. Click the green _Code_ dropdown and under the _SSH_ tab,
+   copy the URL to your clipboard.
+
+## Cloning your Fork
+
+Let’s get the repo down to your local workspace:
+
+```bash
+cd ~
+git clone <your-fork-url-from-clipboard>
+cd symfony
+```
+
+Now we’re going to add the original Symfony repo as an _upstream_ remote. Why?
+So we can easily pull in the latest Symfony updates and stay fresh!
+
+```bash
+git remote add upstream git@github.com:symfony/symfony.git
+```
+
+Let’s grab the latest changes from Symfony (you’ll want to run this command
+whenever Symfony gets an update, so it’s good to get used to it).
+
+```bash
+git fetch upstream
+```
+
+## Use your Reproducer to Fix the Bug
+
+In Part 1, you picked the Symfony version for your reproducer. Now it’s time
+to match that version in our forked Symfony repo. Let’s check it out:
+
+```bash
+git checkout 7.1
+```
+
+Boom! You’ve got the right version. Now, let’s make sure we’re up-to-date
+by merging in any new changes:
+
+```bash
+git merge upstream/7.1
+```
+
+Great! Now, we’re going to create a new branch for our bug fix:
+
+```bash
+git checkout -b my-fix
+```
+
+***TIP
+The branch name (`my-fix`) isn't important, but it should be descriptive and
+unique to your fork.
+***
+
+## Linking your Reproducer to your Fork
+
+Let’s link that reproducer you created to your freshly forked Symfony repo.
+Head back to your reproducer directory:
+
+```bash
+cd ~/my-reproducer
+```
+
+Symfony comes with a handy PHP script to link itself to a project. This will
+symlink all your app’s `symfony/*` packages to your local fork. Super neat, right?
+
+```bash
+php ../symfony/link .
+```
+
+## Fixing the Bug
+
+Open your reproducer in your favorite IDE. Normally, messing with files in the 
+`vendor/` directory is a no-no, but in this case, for files in `vendor/symfony/*`,
```

Comment:
```suggestion
`vendor/` directory is a no-go, but in this case, for files in `vendor/symfony/*`,
```
?

Reply:
I feel like *no-go* has a different meaning. Maybe this doesn't translate well to non-english speakers (or is too _childish_ 😆)?

Reply:
Aha, that exists in my lang too... though, IMO no-no is just a childish version of no-go, so monthly both have the same meaning for me 😅  I'm fine with any option here actually, just wanted to catch some possible misprints :)